CHANGELOG.md

```diff
@@ -33,6 +33,7 @@
   (#465, #469, #474)
 - Linting improvements: use mypy + use ruff-format instead of black for formatting
   (#478, #479)
+- If an output_dir doesn't exist yet, avoid processing being done before an error is raised... #521
```

Comment:
```suggestion
- If an output_dir doesn't exist yet, avoid processing being done before an error is raised... (#518)
```

By convention, if there is an "issue", the link (id) to the issue is put here as the issue typically contains a more user-friendly explanation of what is implemented. If there is only a PR, the id of the PR is put here. The id is/are put between parentheses.